beacon-chain/sync/service.go

```diff
@@ -202,6 +203,8 @@ func NewService(ctx context.Context, opts ...Option) *Service {
 	}
 	// Initialize signature channel with configured limit
 	r.signatureChan = make(chan *signatureVerifier, r.cfg.batchVerifierLimit)
+	// Initialize KZG channel for worker pool
+	r.kzgChan = make(chan *kzgVerifier, 100)
```

Comment:
I think adding a little rationale explaining how this `100` has been designed could be useful.